pkg/middleware/dashboard_redirect_test.go

```diff
@@ -29,8 +29,8 @@ func TestMiddlewareDashboardRedirect(t *testing.T) {
 		})
 
 		sc.handlerFunc = redirectFromLegacyDashboardURL
-		sc.m.Get("/dashboard/db/:slug", sc.defaultHandler)
-		sc.fakeReqWithParams("GET", "/dashboard/db/dash?orgId=1&panelId=2", map[string]string{}).exec()
+		sc.m.Get("/dashboards/uid/:uid", sc.defaultHandler)
```

Comment:
Think these tests might be obsolete if you remove the legacy redirect middlewares and can be removed since no longer used?